packages/app-project/src/shared/components/SubjectPicker/helpers/searchParams.js

```diff
@@ -1,10 +1,10 @@
 export default function searchParams(data) {
-  let query = ''
+  let query = []
   Object.entries(data).forEach(([key, value]) => {
     if (value !== '') {
-      query += `@${key}:${value}*`
+      query.push(`${key}__contains=${value}`)
     }
   })
-  const urlParams = (query !== '') ? `filter_field=${query}` : ''
+  const urlParams = (query !== '') ? query.join('&') : ''
```

Comment:
`query` is now an array now, so the conditional will always evaluate to true. That said, if the query is an empty array, then `[].join('&')` will always evaluate to an empty string `""` anyway, so nobody will notice the difference, haha.

Recommendation is to just change this to `const urlParams = query.join('&')` , just to make the code cleaner.

Reply:
Good point, I'll fix that.